string      rover_type
string[]    rover_names
float64[]   x
float64[]   y
float64[]   z
